pkg/config/config.go

```diff
@@ -128,9 +128,34 @@ func NewConfig(configPath string) (*Config, error) { // nolint // function will
 		}
 	}
 
+	err = cfg.Validate()
+	if err != nil {
+		return nil, errors.Wrap(err, "config validation failed")
+	}
+
 	return cfg, nil
 }
 
+func (cfg *Config) Validate() error {
+	if cfg.SCIM.UserObjectType == "" {
+		return errors.New("scim.user_object_type is required")
+	}
+	if cfg.SCIM.IdentityObjectType == "" {
+		return errors.New("scim.identity_object_type is required")
+	}
+	if cfg.SCIM.IdentityRelation == "" {
```

Comment:
Should also validate that the object in `IdentityRelation` is either `IdentityObjectType` or `UserObjectType`.